web_querier/test_querier.py

```diff
@@ -0,0 +1,24 @@
+from web_querier import Querier
+import pytest
+
+
+class TestQuerier(object):
+    """Test Querier class."""
+
+    def test_valid_url(self):
+        """Test that valid url is accepted."""
+        q = Querier("https://neurosynth.org/api/locations/{}_{}_{}_1/compare/")
+        assert q.url == "https://neurosynth.org/api/locations/{}_{}_{}_1/compare/"
+
+
+    def test_no_room_for_coordinates(self):
+        """Test that url has room for coordinates."""
+        with pytest.raises(ValueError):
```

Comment:
we can use a more specific exception - Instead of raising a generic ValueError, we can raise a more specific exception, such as InvalidURLError